File: SQL/Select/SQLSelectGetDate.vb

' ___________________________________________________
'
'  © Hi-Integrity Systems 2010. All rights reserved.
'  www.hisystems.com.au - Toby Wicks
' ___________________________________________________
'

Option Strict On
Option Explicit On

Namespace SQL

    Public Class SQLSelectGetDate
        Inherits SQLSelectExpression

        Public Sub New()

            MyBase.New(New SQLGetDateFunctionExpression)

        End Sub
         
    End Class

End Namespace

File: SQL/Select/SQLSelectGetDate.cs
// ___________________________________________________
//
//  © Hi-Integrity Systems 2010. All rights reserved.
//  www.hisystems.com.au - Toby Wicks
// ___________________________________________________
//

using System.Collections;
using System;
using System.Data;

namespace DatabaseObjects.SQL
{
	public class SQLSelectGetDate : SQLSelectExpression
	{
		public SQLSelectGetDate() 
            : base(new SQLGetDateFunctionExpression())
		{
		}
	}
}
